gammapy/estimators/flux_map.py

```diff
@@ -27,17 +22,29 @@
     "likelihood": ["norm"],
 }
 
-#TODO: add an entry for is_ul?
+
+# TODO: add an entry for is_ul?
 OPTIONAL_MAPS = {
     "dnde": ["dnde_err", "dnde_errp", "dnde_errn", "dnde_ul"],
     "e2dnde": ["e2dnde_err", "e2dnde_errp", "e2dnde_errn", "e2dnde_ul"],
     "flux": ["flux_err", "flux_errp", "flux_errn", "flux_ul"],
     "eflux": ["eflux_err", "eflux_errp", "eflux_errn", "eflux_ul"],
-    "likelihood": ["norm_err", "norm_errn", "norm_errp","norm_ul", "norm_scan", "stat_scan"],
+    "likelihood": [
+        "norm_err",
+        "norm_errn",
+        "norm_errp",
+        "norm_ul",
+        "norm_scan",
+        "stat_scan",
+    ],
 }
 
+COMMON_MAPS = ["ts", "sqrt_ts"]
```

Comment:
How about other maps: `counts`, `background`, `excess`. 
`ExcessMapEstimator` could be adapted to return flux maps as well but we would need other diagnostics maps in that case.
Shouldn't we leave the flexibility of using any other map?

Reply:
This was the main question I had as well. In general I would prefer to declare somehow which other maps are allowed as well, but this requires to define first which those are. I think defining and documenting the additional quantities e.g. a `ExcessMapEstiamtor` returns is a good thing. And this requires also clarifying the terms we use. E.g. especially in the context of the `ExcessMapEstimator` this is not 100% clear to me yet. E.g. currently we return `counts` and `excess` and `background`, while in the context of an `Estimator` I would rather refer to `npred`, `npred_signal` and `npred_background`.

As a first start I could open an PR extending the documentation here https://docs.gammapy.org/dev/estimators/index.html, for quantities we are currently missing.

Reply:
Following #3278 , I added `npred`, `npred_excess` and `npred_null` to the maps. However it might still be incomplete. I think the next step is to check the result quantities of the current estimators and make a PR to unify it and extend for missing quantities. I'm completely open to allow for any other additional quantity, however for this PR I propose to just stick with the defined and documented ones.